### PR TITLE
Feat: JS plain with class

### PR DESCRIPTION
**Pull request by Amrhub**
- Used `class` instead of `object` and `array`
- Create class methods to add and remove books instead of separate functions for objects.
- fixed all bugs and it's working just as intended

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -1,45 +1,54 @@
+/* eslint-disable comma-dangle */
```
Comment (carlos-ssh) [inline scripts/index.js:1]: - Improve your code and don't use lot of rules in your ESLint. Try to fix the issues.

File: scripts/index.js [hunk 2]
```diff
@@ -1,45 +1,54 @@
...
+  }
 }
 
 form.onsubmit = () => {
   const title = form.elements.title.value;
   const author = form.elements.author.value;
-  if (books.length) {
-    const bookId = books[books.length - 1].id + 1;
-    addBook(title, author, bookId);
+  if (allBooks.books.length) {
```
Comment (carlos-ssh) [inline scripts/index.js:34]: - Kindly fix this error:
Screen Shot 2021-10-19 at 18 36 49

File: scripts/index.js [hunk 3]
```diff
@@ -1,45 +1,54 @@
+/* eslint-disable comma-dangle */
 /* eslint-disable no-use-before-define */
 /* eslint-disable object-shorthand */
 /* eslint-disable no-undef */
+const booksLocalStorage = JSON.parse(localStorage.getItem('books') || '[]');
```
Comment (carlos-ssh) [inline scripts/index.js:5]: - I can't add books into the array i have this error kindly work on the requested changes:
Screen Shot 2021-10-19 at 21 36 37

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Amrhub, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

changesrequested


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/index.js (hunk 3 above)
Comment (Amrhub) [reply scripts/index.js:5]: I am really frustrated with these reviews Its working on my end

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**